climada/entity/exposures/crop_production.py

```diff
@@ -638,6 +747,14 @@ def normalize_several_exp(input_dir=INPUT_DIR, output_dir=OUTPUT_DIR,
             exp_tot_production(list): Exposure crop production value per country
                 (before normalization)
     """
+    if not input_dir:
+        input_dir = INPUT_DIR
+    if not output_dir:
+        output_dir = OUTPUT_DIR
+    if not unit:
+        unit = 't'
+    if yearrange is None:
+        yearrange = (2008, 2018)
```

Comment:
Why `(2008, 2018)`? Default constants should be set in constants or in the config file.